common/djangoapps/student/tests/test_email.py

```diff
@@ -332,8 +333,8 @@ def test_duplicate_activation_key(self):
         user2 = UserFactory.create(email=self.new_email, password="test2")
 
         # Send requests & ensure no error was thrown
-        self.assertIsNone(self.do_email_change(self.user, user1_new_email))
-        self.assertIsNone(self.do_email_change(user2, user2_new_email))
+        self.do_email_change(self.user, user1_new_email)
+        self.do_email_change(user2, user2_new_email)
```

Comment:
Thanks for cleaning up things like this.  I'm constantly finding odd superstitious code in tests....